.buildkite/release-builds.yml

```diff
@@ -14,22 +14,22 @@ common_params:
 
 steps:
 
-  - label: ":wordpress::testflight: WordPress Release Build (App Store Connect)"
+  - label: ":wordpress: :testflight: WordPress Release Build (App Store Connect)"
     command: ".buildkite/commands/release-build-wordpress.sh $BETA_RELEASE"
     env: *common_env
     plugins: *common_plugins
     notify:
     - slack: "#build-and-ship"
 
   # There is no App Center emojiy
```

Comment:
Just realized this comment was there… should have used the occasion to remove it now that it's not true anymore 🙃

Reply:
Follow-up: https://github.com/wordpress-mobile/WordPress-iOS/pull/19283